tests/test_channels.py

```diff
@@ -298,3 +298,31 @@ def test_http_server_auth_token_in_defaults(http_server_auth_token):
             condarc.write_text(condarc_contents)
         else:
             condarc.unlink()
+
+
+def test_local_spec():
+    "https://github.com/conda/conda-libmamba-solver/issues/398"
+    env = os.environ.copy()
+    env["CONDA_BLD_PATH"] = str(DATA / "mamba_repo")
+    process = conda_subprocess(
+        "create",
+        "-p",
+        _get_temp_prefix(use_restricted_unicode=on_win),
+        "--dry-run",
+        "--solver=libmamba",
+        "--channel=local",
+        "test-package",
+        env=env,
+    )
+    assert process.returncode == 0
+
+    process = conda_subprocess(
+        "create",
+        "-p",
+        _get_temp_prefix(use_restricted_unicode=on_win),
+        "--dry-run",
+        "--solver=libmamba",
+        "local::test-package",
```

Comment:
You need `--channel=local` here as well.

Reply:
Nvm, that shouldn't be needed

Reply:
Technically we should not need it. And even if I add it, it doesn't work because `CONDA_BLD_PATH` is ignored by `libmamba`'s hardcoded `local` definition. See https://github.com/mamba-org/mamba/pull/3056#issuecomment-1851576025